Jenkinsfile

```diff
@@ -25,10 +38,10 @@ node {
 
     stage('Build Packages') {
         nodeEnv.inside("-e HOME=${workspace}") {
-          sh "make SETTINGS_FILE=settings/chrome-stage.json dist/chrome-stage.zip"
-          sh "make SETTINGS_FILE=settings/chrome-prod.json dist/chrome-prod.zip"
-          sh "make SETTINGS_FILE=settings/firefox-stage.json dist/firefox-stage.xpi"
-          sh "make SETTINGS_FILE=settings/firefox-prod.json dist/firefox-prod.xpi"
+          sh "make SETTINGS_FILE=settings/chrome-stage.json dist/${gitVersion}-chrome-stage.zip"
+          sh "make SETTINGS_FILE=settings/chrome-prod.json dist/${gitVersion}-chrome-prod.zip"
+          sh "make SETTINGS_FILE=settings/firefox-stage.json dist/${gitVersion}-firefox-stage.xpi"
+          sh "make SETTINGS_FILE=settings/firefox-prod.json dist/${gitVersion}-firefox-prod.xpi"
```

Comment:
The existing `tools/deploy` script expects the `$BROWSER-$VERSION.zip` files to have a prefix. Any prefix could have been used, but using the Git version makes it obvious which commit the package came from.